inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -549,40 +574,10 @@ public function check_job_status( int $id ) {
 		}
 
 		if (
-			200 !== (int) $job_details['code']
-			||
-			empty( $job_details['contents'] )
-			||
-			! isset( $job_details['contents']['shakedCSS'] )
+			200 !== $job_details['code']
```

Comment:
`$job_details['code']` was typecasted to `int` to resolve an error when the value was not an `int`:
https://github.com/wp-media/wp-rocket/pull/6257/commits/24982823e54cf576505cfc70806952bd8a115762

Is there a reason this was removed here?

Reply:
No reason, I didn't pay attention, my bad, I've added it back.